oneanddone/base/static/css/one-and-done.less

```diff
@@ -30,6 +30,11 @@ h4, .billboard h4 {
 p {
     line-height: 1.5;
 }
+pre,
+code {
```

Comment:
This looks pretty good, @tessie. I suggest making a few changes:

1. Remove the italic. With the highlighting it is unnecessary.
2. Make the colour a bit lighter. It seems to be a bit too bright right now.
3. You are overriding the `pre` and `code` class from sandstone, so you should probably bring in the properties from that file too. Specifically the ones at https://github.com/mozilla/oneanddone/blob/master/oneanddone/base/static/css/sandstone/sandstone-resp.less#L229